### ISSUE TITLE
Mastodon iPhone app is partly in russian 

### ISSUE THREAD

**Issue by x-1xps** [mastodon/mastodon-ios#762]
### Steps to reproduce the problem

Open the app

### Expected behaviour

App should only be in english

### Actual behaviour

It is partly in english and partly in russian 

### Detailed description

![FDA3061D-3A1F-4C72-8150-BFCC8391AAF1](https://user-images.githubusercontent.com/118946228/203671303-8e84bb2a-da98-4bb1-bb06-c964dfd41495.png)

### Specifications

Mastodon for iPhone and iPad 1.4.7

**Comment by Gargron** [comment]
Is your system language Russian?

**Comment by x-1xps** [comment]
@Gargron No, it is in danish. I have the russian keyboard installed though, amongst keyboards of other languages

**Comment by steakpinball** [comment]
I think I am experiencing this as well. But for me the incorrect language is Finnish. My system language is English, and I have no keyboard or anything else in Finnish.

Mastodon for iOS 1.4.9 (224)
iOS 16.2 (20C65)

![image](https://user-images.githubusercontent.com/11620342/210234269-a77a3608-02b2-4fcb-91e0-8ba7b93b5593.jpeg)

**Comment by j-f1** [comment]
@steakpinball That’s #824 which will be fixed in the next update.
